homeassistant/components/surepetcare/manifest.json

```diff
@@ -1,8 +1,9 @@
 {
-  "domain": "surepetcare",
-  "name": "Sure Petcare",
-  "documentation": "https://www.home-assistant.io/integrations/surepetcare",
-  "codeowners": ["@benleb", "@danielhiversen"],
-  "requirements": ["surepy==0.7.0"],
-  "iot_class": "cloud_polling"
-}
+    "domain": "surepetcare",
+    "name": "Sure Petcare",
+    "documentation": "https://www.home-assistant.io/integrations/surepetcare",
+    "codeowners": ["@benleb", "@danielhiversen"],
+    "requirements": ["surepy==0.7.0"],
+    "iot_class": "cloud_polling",
+    "config_flow": false
+  }
```

Comment:
Any relevant changes here?

Reply:
`"config_flow": false`

is new, as I just changed it to `false` instead of removing it. its planned to switch it back to yes in a follow-up pr for the config flow

Reply:
Then I suggest you revert the changes in manifest.json

Reply:
Ok, I suggest you give a reason why or a link to the part of the docs which defines that `config_flow` always has to be `True` 🤨 Just for my understanding of course... and to not make the same mistakes again in future...^^

Reply:
As you have been told before, it would be great if you could remove all unrelated and unnecessary changes
https://developers.home-assistant.io/docs/creating_component_code_review/#5-make-your-pull-request-as-small-as-possible

Reply:
reverted